OpenRA.Game/Game.cs

```diff
@@ -173,6 +173,15 @@ internal static void StartGame(string mapUID, WorldType type)
 			GC.Collect();
 		}
 
+		/**
+		 * restart current mission / game
+		 */
```

Comment:
This adds 3 lines of commentary which essentially just repeats the function name.